src/Microsoft.AspNet.Tools.PublishIIS/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+
+namespace Microsoft.AspNet.Tools.PublishIIS
+{
+    public class Program
+    {
+        public static int Main(string[] args)
+        {
+            var app = new CommandLineApplication
+            {
+                // TODO: This needs to be updated once we know how this is going to be called
+                Name = "dotnet(????) publish-iis",
+                FullName = "Asp.Net IIS Publisher",
+                Description = "IIS Publisher for the Asp.Net web applications",
+            };
+            app.HelpOption("-h|--help");
+
+            var publishFolderOption = app.Option("--publish-folder", "The path to the publish output folder", CommandOptionType.SingleValue);
+            var appNameOption = app.Option("--application-name", "The name of the application", CommandOptionType.SingleValue);
+
+            app.OnExecute(() =>
+            {
+                var publishFolder = publishFolderOption.Value();
+                var appName = appNameOption.Value();
+
+                if (publishFolder == null || appName == null)
+                {
+                    app.ShowHelp();
+                    return 2;
+                }
+
+                XDocument webConfigXml = null;
+                var webConfigPath = Path.Combine(publishFolder, "wwwroot", "web.config");
```

Comment:
wwwroot can be renamed via the webroot parameter in hosting.json or command line.